pinot-common/src/main/java/org/apache/pinot/common/metadata/controllerjob/ControllerJobType.java

```diff
@@ -20,6 +20,6 @@
 
 public enum ControllerJobType {
   RELOAD_SEGMENT,
-  RELOAD_ALL_SEGMENTS,
```

Comment:
Do we merge `RELOAD_ALL_SEGMENTS` into `RELOAD_SEGMENT`? Some changes seems unrelated to the current PR topic

Reply:
We are creating a separate znode per jobType and decided to combine reload/reload_all into znode. Now there are totally 3 znodes (one per jobType -> reload, force_commit, rebalance).

Reply:
Lets do this in a separate PR as discussed.